post-processor/vsphere/post-processor_test.go

```diff
@@ -50,24 +71,33 @@ func TestEscaping(t *testing.T) {
 	cases := []escapeCases{
 		{`this has spaces`, `this%20has%20spaces`},
 		{`exclaimation_!`, `exclaimation_%21`},
-		{`hash_#_dollar_$`, `hash_%23_dollar_%24`},
```

Comment:
This test is useful because it validates that there's a big difference between which vars were being encoded before versus which are being encoded now.